app/handler/commands.js

```diff
@@ -803,6 +804,35 @@ exports.handleMessage = function (steamID, message) {
             }
             client.chatMessage(steamID, 'Price check has been requested, the item will be checked.');
         });
+    } else if (isAdmin && command === 'name') {
+        // This has already been used but since I'm planning on rewriting this for user extensions it will remain.
+        const newName = removeCommandFromMessage(message, command);
+
+        community.editProfile({
+            name: newName
+        }, function (err) {
+            if (err) {
+                log.warn('Error while changing name: ', err);
+                client.chatMessage(steamID, 'Error while changing name: ' + err.message);
+
+                return;
+            }
+
+            client.chatMessage(steamID, 'Successfully changed name.');
+        });
+    } else if (isAdmin && command === 'avatar') {
+        const imageUrl = removeCommandFromMessage(message, command);
```

Comment:
Should check if the message contains an image, and if the image is a link to a website